transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollChannelOption.java

```diff
@@ -26,6 +26,9 @@
 
     public static final ChannelOption<Boolean> SO_REUSEPORT = valueOf("SO_REUSEPORT");
 
+    public static final ChannelOption<EpollDomainSocketReadMode> DOMAIN_SOCKET_READ_MODE =
```

Comment:
One question would be: should it be at `io.netty.channel.socket` or do we need a dedicated package for it, such as `io.netty.channel.unix`?